### ISSUE TITLE
Opening a PR, removing 'in-progress' label: Remove self-assignment from issue too?

### ISSUE THREAD

**Issue by mk4111** [dwyl/contributing#89]
When we work on an issue, the first thing we do (upon getting the green light to actually work on the issue), is a) assigning oneself to the task/issue and b) sticking the 'in-progress' label on.

When we are done working on the issue, and have made a PR, we assign the PR to another person to review, and we also remove the 'in-progress' label from the issue, but do we also remove ourselves from 'Assignees'?

**Comment by ghost** [comment]
@mk4111 yes I believe so, then if the PR requires changes the reviewer assigns you back on - @nelsonic and I are going over the contributing guide tomorrow afternoon to help make it easier to read and understand

**Comment by mk4111** [comment]
@markwilliamfirth - cool! Thank you! I'll close this issue once I've had a read of the update!! Hope that's ok! :)

**Comment by mk4111** [comment]
Hi @markwilliamfirth! Forgive me if I'm being a bit of a Humpty Dumpty and it's already been updated, but can you let me know if/when the README gets updated so I can be sure to check it out and close this issue? Thank you!

**Comment by ghost** [comment]
It hasn't been updated yet - it got pushed back :(
I'll update here when it is
